content/en/docs/quickstart.md

```diff
@@ -17,7 +17,21 @@ Before you begin, you need:
 Create and run an OCI-compatible registry on your development computer using Docker and the [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry) container image. The following command creates a registry that is accessible at `localhost:5000`.
 
 ```console
-docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.2
+docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.3
+```
+
+> NOTE:
+> Since v1.0.0-rc.1 release, by default, Notation stores signatures using OCI artifact Manifest, which is defined in [OCI Image spec v1.1.0](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc2)). If you choose a different registry, make sure the registry is OCI Image spec v1.1.0 compliant.
+
+### Building OCI-compatible registry image from source
+
+In some cases, such as if your development computer is running an Apple silicon processor, you can download the source of the image for the OCI-compatible registry. The following example clones the repository, builds the image using `docker buildx`, and creates a registry that is available at `localhost:5000`.
+
+```console
+git clone https://github.com/oras-project/distribution.git
+cd distribution
+docker buildx build -t oras-project/registry:v1.0.0-rc.3 . 
```

Comment:
IMO, the build process is redundant here. It causes the Quick Start guide not to be quick and simple at all.

Reply:
I also think it could be too much for a quick start. We can remove this section once we know the reason why `registry:v1.0.0-rc.2` works but not `registry:v1.0.0-rc.3`.

Reply:
@FeynmanZhou @yizha1 if a new image built for the apple silicon architecture can be created near-term, i agree we should pull this. If not, this needs to be included since the quickstart doesn't work for apple silicon macs as is.

Reply:
The distribution team is working on a new image that supports multi-arch. So two options: 
- Skip the `build registry` section for now, and add it back if needed
- Put `build registry` under the bottom of quick start as additional reference